### DISCUSSION TITLE
Allow setting board column to milestone

### DISCUSSION THREAD

**Discussion by The-Noah** [community/community#16954] [Projects and Issues]
It'd be awesome if when using the board view you can set the column to the milestone.

![image](https://user-images.githubusercontent.com/35819660/169426704-ff8aca7d-48fb-40ac-b1d4-9792a26310ed.png)
![image](https://user-images.githubusercontent.com/35819660/169426731-8fbda40b-e342-474e-be66-2fa3755c5da7.png)

**Comment by meetsussie** [comment]
> It'd be awesome if when using the board view you can set the column to the milestone.
> 
> ![image](https://user-images.githubusercontent.com/35819660/169426704-ff8aca7d-48fb-40ac-b1d4-9792a26310ed.png) ![image](https://user-images.githubusercontent.com/35819660/169426731-8fbda40b-e342-474e-be66-2fa3755c5da7.png)

Yes it will be very awesome
